azure-pipelines.yml

```diff
@@ -101,6 +101,24 @@ steps:
       [[ "$PYTHON_VERSION" = 'Pre' ]]
   displayName: 'pytest'
 
+- bash: |
+    case "$(python -c 'import sys; print(sys.platform)')" in
+    linux)
+      bash <(curl -s https://codecov.io/bash)  -f "!*.gcov" -X gcov -y ci/codecov.yml
+      ;;
+    darwin)
+      bash <(curl -s https://codecov.io/bash) -f "!*.gcov"
+      ;;
+    win32)
+      choco install codecov
```

Comment:
Try `pip install codecov` like we do on Appveyor, choco seems to be installing `codecov-exe`